### DISCUSSION TITLE
Doubts about usage

### DISCUSSION THREAD

**Discussion by gabriellatavares** [dm4t2/vue-currency-input#292] [General]
Hello! I'm migrating a Vue 2 app to Vue 3 and I was wondering if I can use vue-currency-input outside of an input? For example, I have filter in the globalProperties and I wish to convert some currencies accordingly to the user preference, and those currencies are all over the app. Can I still use this package? The Vue 2 version uses vue-currency-filter for that. 

Thanks!

**Comment by dm4t2** [comment]
No, see #163. But you can simply use [Intl.NumberFormat.prototype.format()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/NumberFormat/format) for printing currency formatted numbers. There's no need for a library.
